client/eslintrc.json

```diff
@@ -0,0 +1,22 @@
+{
```

Comment:
I think your linting did not apply correctly because of 2 configurations (one for v9+ one for v8+). we only need a v8 config. Also, you used a eslintrc.json config instead of .js, which is also fine, but it means you are not paying attention to the details in the instructions of the task, where it was stated to use eslintrc.js.

Maybe we still need to tweak the eslint config, but let's first clean up the redundant package dependencies+files. Also, revert the src/* changes so that we have a clean state as main in src for now